framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1259,17 +1097,17 @@ wifi_manager_result_e _handler_on_reconnect_state(_wifimgr_msg_s *msg)
 	}
 #else /* WIFIDRIVER_SUPPORT_AUTOCONNECT*/
 	if (msg->event == EVT_DISCONNECT) {
-		ndbg("[WM] AUTOCONNECT fail: go to DISCONNECTED\n");
-		_close_ipaddr_dhcpc();
+		nvdbg("[WM] AUTOCONNECT fail: go to DISCONNECTED\n");
```

Comment:
```
nvdbg("[WM] AUTOCONNECT state : %d\n", msg->event);
if()
else if()
...

Reply:
Your suggestion looks better.
Applied. Thanks for the comment.